src/openforms/contrib/digid_eherkenning/signals.py

```diff
@@ -0,0 +1,19 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from digid_eherkenning.models.digid import DigidConfiguration
+from digid_eherkenning.models.eherkenning import EherkenningConfiguration
```

Comment:
you can import both models from `models` module (this is a pattern that you should always follow)